File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ViacepAddressLookup SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ViacepAddressLookupMakeContext
{
    public static function call(array $ctxmap, ?ViacepAddressLookupContext $basectx): ViacepAddressLookupContext
    {
        return new ViacepAddressLookupContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# ViacepAddressLookup SDK utility: make_context
require_relative '../core/context'
module ViacepAddressLookupUtilities
  MakeContext = ->(ctxmap, basectx) {
    ViacepAddressLookupContext.new(ctxmap, basectx)
  }
end
